nservicebus/security/encryption.md

```diff
@@ -96,12 +108,19 @@ This allows you to explicitly handled the encryption and decryption of each valu
 
 ## Message Encryption
 
-Message encryption leverages the pipeline to apply encryption to the whole message body.
+Message encryption encrypts the complete message body. It can even include the message headers if these would contain sensitive data.
```

Comment:
@DavidBoike message encryption is mentioned to list the encryption possibilities. I agree that reading this should make the user aware that we do not offer this out of the box but that we have a sample to demonstrate that it is possible to implement message encryption.

Regarding Service Control, yes it is interesting, as it cannot do anything with a fully encrypted message without knowing how to decrypt it. I don't know enough of SC to write something correct about its behavior.